packages/devp2p/package.json

```diff
@@ -37,6 +37,7 @@
     "tsc": "../../config/cli/ts-compile.sh",
     "lint": "../../config/cli/lint.sh",
     "lint:fix": "../../config/cli/lint-fix.sh",
+    "tape": "tape -r ts-node/register",
```

Comment:
I think I first encountered the `tape` script in the `client` package a while back and found it very useful.  I really like it as a shorthand for being able to run an individual test file for those instances when I'm either triaging an existing test that is broken because of code changes I've made or else where I'm constructing a new one and I don't want to run the entire test suite for a package every time I need to re-run a test.  It's just a convenience thing for me as typing `npm run tape -- 'test/whatever_test_script_I_need_to_run.spec.ts'`  is easier than `npx tape -r ts-node/register 'test/...`  It's not original to me and existed at least in the client package  so I've just been adding it to packages that I work on when I discover that it's not there.  `npm run test` is a different use case as it always runs the complete test suite for the the package where it's executed from.

All that to say, I don't have a strong opinion on this if you feel strongly otherwise and can just easily set up a local snippet in my own dev environment.

Reply:
yes I find it helpful to have as well, we should update the test script to use it so it doesn't go unused and dedupes code (so for devp2p `"test": "npm run tape --  ./test/index.ts"`)

@holgerd77 it was originally introduced here https://github.com/ethereumjs/ethereumjs-client/pull/169 - I've found it really helpful in running individual test files during development and have added it in a few other packages since.

Reply:
Ok, can go along, have answered in the chat as well on this.